### ISSUE TITLE
Question about inference function in moco/builder.py

### ISSUE THREAD

**Issue by pseulki** [dvlab-research/Parametric-Contrastive-Learning#13]
Hello, I'm impressed with your work and thank you for sharing the codes.

I have a question abut inference function in  LT/moco/builder.py  !
In the code below, you are not using _q_ but using _self.feat_after_avg_q_ as input for linear.
Then, it doesn't seem to need the first two lines in the code.
Or, should _self.feat_after_avg_q_ be changed to _q_?
Can you check this?
Thank you in advance! :) 

_def _inference(self, image):
        q = self.encoder_q(image)
        q = nn.functional.normalize(q, dim=1)
        encoder_q_logits = self.linear(self.feat_after_avg_q)

        return encoder_q_logits_

**Comment by jiequancui** [comment]
Hi, 

Thanks for your question.

"self.feat_after_avg_q" is updated by the hook function defined in https://github.com/dvlab-research/Parametric-Contrastive-Learning/blob/main/LT/moco/builder.py #95 

Best,
Jiequan
